### PR TITLE
Add new contributors from the core team.

### PR DESCRIPTION
**Pull request by ttoine**

### CODE REVIEW

File: coreteam.py [hunk 1]
```diff
@@ -36,7 +36,9 @@
     'boubkerbribri',      # Boubker B.
     'matthieu-rolland',   # Matthieu R.
     'Progi1984',          # Franck L.
-    'Robin-Fischer-PS',   # Robin F
+    'Robin-Fischer-PS',   # Robin F.
+    'samuel-pires',       # Samuel S.
```
Comment (matks) [inline coreteam.py]: Samuel S ? Isn't it Samuel P ? 😛

Comment (ttoine) [reply coreteam.py]:
```suggestion
    'samuel-pires',       # Samuel P.
```
You are right. My bad.

**Comment by ttoine** [comment]
@PierreRambaud suggestion applied.

**Review by PierreRambaud — APPROVED** [review]

File: coreteam.py [hunk 2]
```diff
@@ -36,7 +36,9 @@
     'boubkerbribri',      # Boubker B.
     'matthieu-rolland',   # Matthieu R.
     'Progi1984',          # Franck L.
-    'Robin-Fischer-PS',   # Robin F
+    'Robin-Fischer-PS',   # Robin F.
+    'samuel-pires',       # Samuel P.
+    'atomiix',            # Thomas B.
```
Comment (PierreRambaud) [inline coreteam.py:41]: Could we add @NeOMakinG?

**Review by PierreRambaud — DISMISSED** [review]
One more change :)

File: coreteam.py (hunk 2 above)
Comment (ttoine) [reply coreteam.py:41]: ok let's do it

**Comment by ttoine** [comment]
can someone merge this pr please ?

**Comment by PierreRambaud** [comment]
@ttoine Could you also add @NeOMakinG in the list?

File: coreteam.py (hunk 2 above)
Comment (matks) [reply coreteam.py:41]: Added

**Review by matks — APPROVED** [review]

**Pull request merged**